src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/create_index_pattern_wizard.js

```diff
@@ -114,7 +115,7 @@ uiModules.get('apps/management')
     }
 
     // All system indices begin with a period.
-    return indices.filter(index => !index.name.startsWith('.'));
+    return indices.filter(index => !index.name.startsWith('.')).slice(0, MAX_NUMBER_OF_MATCHING_INDICES);
```

Comment:
This will only apply the cap to queries which exclude system indices. Wouldn't we want to apply it to queries which also include system indices? Something like this:

```javascript
const whiteListIndices = indices => {
  if (!indices) {
    return indices;
  }

  const acceptableIndices =
    this.doesIncludeSystemIndices
    ? indices
    // Exclude system indices, which begin with a period.
    : indices.filter(index => !index.name.startsWith('.'));

  return acceptableIndices.slice(0, MAX_NUMBER_OF_MATCHING_INDICES);
};
```

Reply:
Nice catch!